src/main/java/org/folio/service/mapping/RuleFactory.java

```diff
@@ -80,6 +84,9 @@
   private static final String SET_METADATA_UPDATED_DATE_FUNCTION = "set_transaction_datetime";
   private static final String SET_METADATA_CREATED_DATE_FUNCTION = "set_fixed_length_data_elements";
   private static final String MOD_OF_ISSUANCE_ID_FUNCTION = "set_mode_of_issuance_id";
+  private static final String SET_LOCATIONS_FUNCTION_NEW = "set_locations_function_new"; // a new function for locations, should replace old one
```

Comment:
If we are replacing old one, we'll need a migrations cript

Reply:
ok, so I think we need keep the function name in  generate-marc-utils and the name here the same. Are you going to modify it?

Reply:
Yeah,  I'm modifying it in scope of MDEXP-252